### ISSUE TITLE
Sprint 1 - Drivetrain - Rio and Kai

### ISSUE THREAD

**Issue by riochan26** [sta-titansrobotics/27294-IntoTheDeep#12]
Main objective:

Ideate the main design of the frame
Incorporate the motors into the frame, while leaving room for Mecanum Wheels
Compensate for space needed by the moving arm that will be placed near the front of the robot
Ensure the frame is sturdy and able to hold weight
Ensure the frame and wheels fit within competition regulations

End Goal:
Finish 95% of Drivetrain (Mechanisms working, but both halves of the drivetrain connected with little amounts of support)

**Comment by riochan26** [comment]
October 7th, 2024

Ideating stage is complete, planning on having the motors on the inside of the two side brackets, as shown in the image. Planning on moving the front motors back to compensate for the space that will be used by the moving arm. Also planning on adding supports to ensure it will not break during competition, by using two long 15mm extrusions and using the smaller C-Channels in between to have more points of connection, while also protecting the motors.
![image (1)](https://github.com/user-attachments/assets/1235400c-6a0f-4f89-afff-732b0c65c618)

**Comment by riochan26** [comment]
October 8th, 2024

Motor has been moved back on one side, starting the process of creating the chain loop to redirect the motor's power to the front of the robot. All of the planetary motors have been tested, with one needing to be taken apart and rebuilt. Planning on scrapping the old robots for another motor to finish the build.
![image](https://github.com/user-attachments/assets/2a7561a9-de88-476d-b302-e4edd2e1ace0)

**Comment by riochan26** [comment]
October 9th, 2024

Recreated some key components through the help of the online manual. Kai tested and fixed our faulty third motor, and found a fourth motor to use. I started working on getting the chain to the right length so it can be used in redirecting the motor's power to the front of the drivetrain, while Kai is creating a mirrored version for the other side of the drivetrain. We have generated a couple of ideas on combining the two sides with various supports.
![IMG_3919 (1)](https://github.com/user-attachments/assets/c2e544c7-f5f6-461c-970a-ed48bb03e768)

**Comment by riochan26** [comment]
October 10th, 2024

Both sides of the drivetrain have been created. Chain was too short, so had to make it longer. Drivetrain should either be fully completed or almost fully completed by the end of the sprint. Ideas for how to bring both sides of the drivetrain have been thought of, and will most likely be put into effect tomorrow in class. I realized that my original idea of using the master link for the chains was not going to work well, so I decided to scrap the idea and instead use chains traditionally.
![image](https://github.com/user-attachments/assets/fc1bdd67-cb30-40fc-a33a-2b9abd66c4ed)

**Comment by riochan26** [comment]
October 11th, 2024

As Kai and I were done our respective sides of the drivetrain, we both decided to create the chain together. Unfortunately, we were unable to finish the chain, as we had to make constant readjustments to it. The closest we got to completing the chain is shown in the image below, but it is not fully functional. We tried increasing the size of the chain, but that made it too long, leading to unnecessary slack that could mess up our driving. Due to this shortcoming, we were barely unable to reach our goal for this sprint. If we had a little more experience with how the chains worked and a little extra time, I believe that we could have reached our end goal for this sprint.
![image (1)](https://github.com/user-attachments/assets/bfe8afb5-6977-4000-b8cf-65f87be597eb)
